src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts

```diff
@@ -101,18 +110,36 @@ declare namespace Uno.UI {
             * You need to call addView to connect it to the DOM.
             */
         createContent(contentDefinition: IContentDefinition): string;
+        /**
+            * Create a html DOM element representing a Xaml element.
+            *
+            * You need to call addView to connect it to the DOM.
+            */
+        createContentFast(pParams: number): boolean;
+        private createContentInternal(contentDefinition);
+        /**
+            * Set a name for an element.
+            *
+            * This is mostly for diagnostic purposes.
+            */
+        setName(elementId: number, name: string): string;
         /**
             * Set a name for an element.
             *
             * This is mostly for diagnostic purposes.
             */
-        setName(elementId: string, name: string): string;
+        setNameFast(pParam: number): boolean;
+        private setNameInternal(elementId, name);
         /**
             * Set an attribute for an element.
             */
         setAttribute(elementId: string, attributes: {
             [name: string]: string;
         }): string;
+        /**
+            * Set an attribute for an element.
+            */
+        setAttributeFast(pParams: number): boolean;
```

Comment:
from what I understood, the [...]Fast methods are now the ones that should be used. Instead of naming them with the fast suffix, I would let them use the standard name and suffix the old ones with Slow or legacy.

Reply:
Agreed, this is now updated.